src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -51461,6 +51457,7 @@ static void F(IEnumerable<object[]?[]> source)
         }
 
         [Fact]
+        [WorkItem(35151, "https://github.com/dotnet/roslyn/issues/35151")]
         public void ForEach_19()
```

Comment:
>ForEach_19 [](start = 20, length = 10)

Now that the implementation of nullability analysis for type parameters was simplified, I'm not very worried, but it may still be good to test the generic scenario (if not covered already):

```C#
void M1<TEnumerator>(TEnumerator e) where TEnumerator : IEnumerator_ /* ?/! cases*/
{
    ...
```